src/useradd.c

```diff
@@ -1591,6 +1591,13 @@ static void process_flags (int argc, char **argv)
 		if (getdef_bool ("CREATE_HOME")) {
 			mflg = true;
 		}
+	} else {
+		/* If SYS_USER_AUTO_GROUPS_ENAB is disabled,
+		 * then do not automatically add supplements groups for system users. */
+		if (!getdef_bool ("SYS_USER_AUTO_GROUPS_ENAB") && !Gflg && do_grp_update) {
```

Comment:
Please don't add whitespace here in `getdef_bool (` either.  I know the old code has it, but I'm working on slowly getting rid of that.

Thanks!

Reply:
Thanks!